src/Telegram.Bot/TelegramBotClientExtensions.ApiMethods.cs

```diff
@@ -2849,8 +2868,8 @@ await botClient.ThrowIfNull()
         this ITelegramBotClient botClient,
         ChatId chatId,
         int messageThreadId,
-        string name,
-        string iconCustomEmojiId,
+        string? name = default,
+        string? iconCustomEmojiId = default,
```

Comment:
below `new EditForumTopicRequest(chatId, messageThreadId, name, iconCustomEmojiId),`
should be 
```c#
new EditForumTopicRequest(chatId, messageThreadId)
{
  Name = name,
  IconCustomEmojiId = iconCustomEmojiId,
},
```

Reply:
and fix `param` section:

    /// <param name="name">New topic name, 0-128 characters. If not specififed or empty, the current name of the topic will be kept</param>
    /// <param name="iconCustomEmojiId">
    /// New unique identifier of the custom emoji shown as the topic icon. Use 
    /// <see cref="GetForumTopicIconStickersRequest"/> to get all allowed custom emoji identifiers. Pass an empty string to remove the icon. 
    /// If not specified, the current icon will be kept